release.md

```diff
@@ -0,0 +1,12 @@
+**Design system - build and deploy HOWTO**
+
+To deploy package to the npm you need to be logged in. To confirm that you are logged in use you can use 'npm whoami'. If you are not use npm login and follow instructions.
+
+After that run:
+
+- 'npm run check' - this fires up linter and unit tests that are provided per package
+- 'npm run prettier' - to format code across the packages
+- 'npm run predeploy' - cleans up old dist files and builds new ones
+- 'npm run deploy' - deploys files to npm - accept suggested versioning. **Package won't be versioned if not changed, and if changed package will catchup with other packages in versioning.**
```

Comment:
We could automatically run `predeploy` within `deploy` which would simply reduce the whole process to one operation. What we could simply add to this is a detailed description of what is happening underneath.

I would be also nice to guide the "publisher" through some lerna steps, with screenshots if possible. We could include info on when to release a `pre-release` version and when to follow classic `semver`.

Reply:
👌🏼